pkg/types/installconfig.go

```diff
@@ -83,6 +83,17 @@ const (
 	PolicyAlways PolicyType = "Always"
 )
 
+// UserConfiguredDNS enables use of an external user configured DNS solution in place of the default DNS.
+// +kubebuilder:validation:Enum="";Enabled;Disabled
+type UserConfiguredDNS string
+
+const (
+	// EnableUserDNS allows use of user configured DNS that is not the default Cloud DNS.
+	EnableUserDNS UserConfiguredDNS = "Enabled"
+	// DisableUserDNS continues use of the default Cloud DNS that will be configured by OpenShift.
+	DisableUserDNS UserConfiguredDNS = "Disabled"
```

Comment:
Do we need to know explicitly if it is "Disabled" or can we just assume that it is disabled when Enabled is not true ?